docs/linux/external_fuzzing_usbip.md

```diff
@@ -0,0 +1,34 @@
+# **USB/IP Fuzzing for Linux Kernel**
+
+Syzkaller supports fuzzing the Linux kernel USB/IP subsystem externally. We can set up a virtual network and send USB/IP packets to the client kernel as they are being received from an external server.  
+
+USB/IP fuzzing needs USB/IP configurations to be enabled. You can find the list in the configurations part. 
+
+USB/IP Fuzzing consists of 3 main parts:
+1. USB/IP pseudo-syscalls
+2. Syzkaller descriptions.
+3. KCOV changes that allow to collect coverage from background threads and interrupts
```

Comment:
`.` at the end of this line.

These don't exist yet, right? Let's have a `Future improvements` section where we can put it for now. And also mention fuzzing the server side of the connection.